src/category_theory/filtered.lean

```diff
@@ -362,6 +375,11 @@ lemma coeq₃_condition₃ {j₁ j₂ : C} (f g h : j₁ ⟶ j₂) :
   f ≫ coeq₃_hom f g h = h ≫ coeq₃_hom f g h :=
 eq.trans (coeq₃_condition₁ f g h) (coeq₃_condition₂ f g h)
 
+lemma span {i j j' : C} (f : i ⟶ j) (f' : i ⟶ j') :
```

Comment:
Maybe deserves a docstring
```
For every span `j <-- i --> j'`, there exists a cone `j --> k <-- j'` such that the square commutes.
```

Reply:
Side comment: this statement is also used as a condition in theorems in Stacks:
https://stacks.math.columbia.edu/tag/09WS
https://stacks.math.columbia.edu/tag/002X